pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -164,10 +166,10 @@ func (m *GracefulTerminationManager) deleteRsFunc(rsToDelete *listItem) (bool, e
 	}
 	for _, rs := range rss {
 		if rsToDelete.RealServer.Equal(rs) {
-			// Delete RS with no connections
-			// For UDP, ActiveConn is always 0
-			// For TCP, InactiveConn are connections not in ESTABLISHED state
-			if rs.ActiveConn+rs.InactiveConn != 0 {
+			// Delete TCP RS with no connections
+			// For other protocols, don't perform graceful termination
+			//   (existing connections will be deleted because sysctlExpireNoDestConn=1)
+			if rsToDelete.VirtualServer.Protocol == strings.ToLower(string(v1.ProtocolTCP)) && rs.ActiveConn+rs.InactiveConn != 0 {
```

Comment:
re: `strings.ToLower(string(v1.ProtocolTCP))`, v1.ProtocolTCP is mainly a type used by the Service API and is only applicable when comparing values in that resource. For comparing protocols from `VirtualServer`, we can probably use a separate constant for this.

Reply:
I personally think just `rsToDelete.VirtualServer.Protocol == "tcp"` is also fine, having a constant may not be necessary either :)

Reply:
I completely agree
I just changed it to "tcp"